packages/bpk-component-nudger/README.md

```diff
@@ -47,19 +49,104 @@ class App extends Component {
 }
 ```
 
+### Configurable `BpkConfigurableNudger`
+
+```js
+import React, { Component } from 'react';
+import BpkLabel from 'bpk-component-label';
+import { BpkConfigurableNudger } from 'bpk-component-nudger';
+
+class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      value: 1,
+    };
+  }
+
+  handleChange = value => {
+    this.setState({ value });
+  };
+
+  compareValues = (a, b): number => {
```

Comment:
```suggestion
  compareValues = (a, b): => {
```

I _think_ we usually keep Flow types out of the example code in readmes.

Reply:
Sure thing, I guess this was more of a good example people could just copy and run with straight away?

Reply:
Yeah totally! But that makes it even more of a reason to follow best practice right? If people are going to copy and use this in their own code we should make sure it's 💯